.gitignore

```diff
@@ -1,3 +1,4 @@
 target/
 .idea
 portfolio/node_modules/
+package-lock.json
```

Comment:
Please add the `node_modules/` here as well. You'll likely need to reapply the .gitignore following the instructions here: https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-file

Cool thing to try out: https://gitignore.io